docs/src/components/TopGamesOnDusk/styles.module.scss

```diff
@@ -0,0 +1,452 @@
+}
+
+.otherGame {
+  margin: 0;
+
+  display: grid;
+  grid-template-columns: 2fr 3fr 12fr 8fr 2fr;
+  gap: 32px;
+  align-items: center;
+
+  position: relative;
+  padding: 0 42px;
+  height: 88px;
+
+  background: linear-gradient(178deg, #2a0057 2.86%, #58009a 71.35%);
+  border-radius: 8px;
+
+  counter-increment: other-games;
+
+  &::before {
+    order: 1;
+    content: counter(other-games) ".";
+    position: relative;
+    justify-self: self-start;
+
+    font-size: 30px;
+    font-weight: 700;
+    line-height: 36px;
+    letter-spacing: -0.02em;
+    text-align: center;
+  }
+
+  .title {
+    margin: 0;
+
+    order: 3;
+    margin-left: 10px;
+
+    font-size: 28px;
+    font-weight: 700;
+    line-height: 36px;
+    letter-spacing: -0.02em;
+
+    color: #eac6ff;
+  }
+
+  .previewImg {
+    order: 2;
+
+    // Devide original dimensions (104x104) by 4 and then scale it up by 4
+    width: 26px;
+    height: 26px;
+    border-radius: 3px;
+    box-shadow: 3.5px 3.5px 2px 0px #20022566;
+    transform: rotate(-3.4deg) scale(4);
+
+    object-fit: cover;
+    object-position: center 20%;
+    position: relative;
+    justify-self: center;
+  }
+
+  .developers {
+    order: 4;
+  }
+
+  .stat {
+    margin: 0;
+
+    order: 5;
+
+    justify-self: self-end;
+
+    font-size: 24px;
+    font-weight: 700;
+    line-height: 30px;
+    letter-spacing: -0.02em;
+
+    background: linear-gradient(180deg, #35f3fa 0%, #0085ff 100%);
+    background-clip: text;
+    -webkit-background-clip: text;
+    -webkit-text-fill-color: transparent;
+  }
+}
+
+.developers {
+  &.inlined {
+    display: flex;
+    flex-wrap: wrap;
+    gap: 8px;
+  }
+
+  > .avatars {
+    /* Reset */
+    > li + li {
+      margin: 0;
+    }
+    margin: 0;
+    padding: 0;
+
+    list-style: none;
+
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    gap: 0; /* Use margin-right because of negative number */
+    flex-wrap: nowrap;
+    flex-shrink: 0;
+
+    > li,
+    > li + li {
+      margin-right: -8px;
+    }
+    > li:last-child {
+      margin-right: 0;
+    }
+
+    > li > img {
+      width: 33px;
+      height: 33px;
+    }
+  }
+
+  > .names {
+    /* Reset */
+    > li + li {
+      margin: 0;
+    }
+    margin: 0;
+    padding: 0;
+
+    list-style: none;
+
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    gap: 3px;
+    flex-wrap: wrap;
+    flex-shrink: 0;
+
+    font-size: 12px;
+    font-weight: 700;
+    line-height: 15.6px;
+    text-align: center;
+
+    color: #ffffff;
+
+    > li::before {
+      content: "@";
+    }
+
+    > li::after {
+      content: ",";
+    }
+
+    > li:last-child::after {
+      content: "";
+    }
+  }
+}
+
+@media screen and (max-width: 1250px) {
+  .topGames {
+    flex-direction: column;
+    align-items: center;
+    gap: 190px;
+
+    margin-top: 190px;
+  }
+
+  .topGame {
+    max-width: 300px;
+    padding-bottom: 20px;
+
+    &:global(.place-1) {
```

Comment:
Not sure why we need a global class for it.

Reply:
I dynamically generate the class name based on `place` prop.
It is to simplify code in react component.

Reply:
Just realized I can do it without global. Done in https://github.com/dusk-gg/dusk/pull/527/commits/3647058a4218bba982860aa969c50400a54a0cb4